python/test/eager_mode/normalize_args_kwargs.py

```diff
@@ -0,0 +1,77 @@
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# Also available under a BSD-style license. See LICENSE.
+
+# RUN: %PYTHON %s | FileCheck %s
+
+
+import torch
+
+from framework import register_test, run_tests
+from torch_mlir.eager_mode.torch_mlir_dispatch import normalize_args_kwargs
+
+
+# CHECK: PASS - should_normalize
+@register_test
```

Comment:
I don't think you need to "register" anything. Just running it immediately is fine. The requrements of the e2e testing framework are a bit different since we need to drive it from the command line, filter tests, etc. But in this case we already have lit which can do that at .py-file granularity which seems to be enough.